### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by DmytroBondariev**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = list()
+    for person in people:
+        name = person["name"]
+        age = person["age"]
+        new_person = Person(name, age)
+        person_list.append(new_person)
```
Comment (Vasyl-Smutok) [inline app/main.py:16]: use list compression here

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,25 @@
...
+    for person in people:
+        if person.get("wife"):
+            Person.people[person["name"]].wife = \
+                Person.people[person["wife"]]
+        if person.get("husband"):
+            Person.people[person["name"]].husband = \
+                Person.people[person["husband"]]
```
Comment (Vasyl-Smutok) [inline app/main.py:23]: use `get` everywhere

File: app/main.py [hunk 3]
```diff
@@ -1,8 +1,25 @@
...
+    person_list = list()
+    for person in people:
+        name = person["name"]
+        age = person["age"]
+        new_person = Person(name, age)
+        person_list.append(new_person)
+    for person in people:
+        if person.get("wife"):
+            Person.people[person["name"]].wife = \
+                Person.people[person["wife"]]
```
Comment (Vasyl-Smutok) [inline app/main.py:20]: don't use `\`

**Review by Vasyl-Smutok — CHANGES_REQUESTED** [review]

**Review by Grayder0152 — CHANGES_REQUESTED** [review]
Pls, resolve comment

File: app/main.py (hunk 3 above)
Comment (Dmytry95) [reply app/main.py:19]: i can`t make my code work without reverse slash
Pycharm puts it automatically when I move the code line

**Review by Grayder0152 — APPROVED** [review]

**Pull request closed without merging**